client/src/components/InterpreterSignUpForm.js

```diff
@@ -158,7 +192,18 @@ class InterpreterSignUpForm extends Component {
                     required
                   />
                 </Form.Field>
-
+                {this.state.hasErrors ? (
+                  <div>
+                    <Message negative>
+                      <Message.Header>
+                        We're sorry, we can't create your account
```

Comment:
According to the ticket, the error message header should be: "An error occurred"